cmake/PackageConfig.cmake.in

```diff
@@ -9,12 +9,14 @@
 #  * @PROJECT_NAME@::@PROJECT_NAME@
 
 # Include targets file.  This will create IMPORTED target @PROJECT_NAME@
+
+# Depends on w3nco and bacio, but users must link their own version
+# by using find_package(w3nco) and find_package(bacio)
+
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
 
 include(CMakeFindDependencyMacro)
 
-find_dependency(bacio CONFIG)
-find_dependency(w3nco CONFIG)
```

Comment:
should'nt these be kept?
these need to be found when the library is being used, otherwise it will fail.

Reply:
Yes